extensions/amp-audio/validator-amp-audio.protoascii

```diff
@@ -13,12 +13,29 @@
 # See the License for the specific language governing permissions and
 # limitations under the license.
 #
-tags: {  # amp-audio
+tags: {  # amp-audio 1.0
+  html_format: AMP
+  tag_name: "SCRIPT"
+  satisfies: "amp-audio 1.0"
+  excludes: "amp-audio 0.1"
+  extension_spec: {
+    name: "amp-audio"
+    version_name: "v1.0"
+    version: "1.0"
+    deprecated_allow_duplicates: true
+    requires_usage: EXEMPTED
```

Comment:
Two things:
- We want to remove `deprecated_allow_duplicates: true ` and `requires_usage: EXEMPTED` in new 1.0 versions.
- We have a new `bento_supported_version` tag that should also be used:
```suggestion
    bento_supported_version: "1.0"
```